### PR TITLE
Created a definition file to build new `.sif` file with singularity

### PR DESCRIPTION
**Pull request by JoshQuaintance**
# Description

The definition file adds the ability to create a new `.sif` container but currently, because of the `bing-image-downloader` the new sif will not work properly until it is removed.

The definition file will update the containers to 3.13 with miniconda and discordpy 2.5.2

## Issues
N/A

## Type of change

Select one or more of the following:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. List any edge cases you tested.
If you come up with any edge cases you didn't test while writing this PR, cancel the PR and test again.

- [x] Created a new `.sif` with new `.def` file
- [x] Tested the dependencies inside the containerd
- [x] Tested startup with the new `.sif`
- [x] Tested each commands and confirmed that they worked
- [x] Tested the class builder
- [x] Tested the class destroyer 

# Checklist:

- [x] All local commits have been pushed to remote
- [x] All changes on the base branch have been merged into this branch, either by rebase or merge
- [x] My code is [PEP-8](https://pep8.org/) compliant (excluding maximum line length, keep that to 100ish characters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added [Google Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for all new functions/methods
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by JoshQuaintance** [comment]
Just made the issue about `bing-image-downloader` in #421

File: discordpy.def [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    mkdir -p /build/miniconda3
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /build/miniconda3/miniconda.sh
+    bash /build/miniconda3/miniconda.sh -b -u -p /build/miniconda3
+    rm /build/miniconda3/miniconda.sh
+
+    # Activate conda environment
+    . /build/miniconda3/bin/activate
+
+    # Give access to the build directory to all users
+    chmod -R o+rwx /build
```
[the remark this answers is not in the document]
Comment (JoshQuaintance) [reply discordpy.def]: I removed the chmod because it is irrelavant now

File: discordpy.def [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    # Initialize conda
+    conda init --all
```
[the remark this answers is not in the document]
Comment (JoshQuaintance) [reply discordpy.def:34]: Nah, L Bozo

File: discordpy.def [hunk 3]
```diff
@@ -0,0 +1,52 @@
+Bootstrap: docker
+From: ubuntu:24.04
+Stage: build
```
[the remark this answers is not in the document]
Comment (JoshQuaintance) [reply discordpy.def:3]: Nah, another L Bozo

**Comment by JoshQuaintance** [comment]
Thanks for nothing copilot

**Review by JakeHamblin — APPROVED** [review]
lgtm, only file additions
